.travis.yml

```diff
@@ -28,7 +30,7 @@ before_script:
 script:
 - docker build --target test --tag todo.app:test .
 - docker run --env-file .env.test todo.app:test tests
-- docker run -e SECRET_KEY -e DATABASE_NAME -e DB_CONNECTION todo.app:test tests_e2e
+- docker run -e SECRET_KEY -e DATABASE_NAME -e DB_CONNECTION -e LOGGLY_TOKEN -e LOG_LEVEL todo.app:test tests_e2e
```

Comment:
We don't really want to tests to send any logs to Loggly. It would just be noise. More useful to have the application logs visible in the pipeline output.